src/Sarif/CodeGenHints.json

```diff
@@ -18,6 +18,14 @@
           "override"
         ]
       }
+    },
+    {
+      "kind": "AttributeHint",
```

Comment:
>AttributeHint [](start = 15, length = 13)

@lgolding FYI, this hint doesn't appear to be operational for a wildcard spec like this one.

Reply:
That's disturbing. I filed Microsoft/JSchema#51, "AttributeHint doesn't work on a wildcard."

---
In reply to: [221087141](https://github.com/Microsoft/sarif-sdk/pull/1103#discussion_r221087141) [](ancestors = 221087141)

Reply:
Now that you know this, I have removed the non-functional hint. I was able to get things to work by providing conversion at a lower level, the SerializedPropertyInfo...

---
In reply to: [221313782](https://github.com/Microsoft/sarif-sdk/pull/1103#discussion_r221313782) [](ancestors = 221313782,221087141)